examples/api/lib/material/app/app.0.dart

```diff
@@ -0,0 +1,71 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+/// Flutter code sample for [MaterialApp].
+
+void main() {
+  runApp(const MaterialAppExample());
+}
+
+class MaterialAppExample extends StatefulWidget {
+  const MaterialAppExample({super.key});
+
+  @override
+  State<MaterialAppExample> createState() => _MaterialAppExampleState();
+}
+
+class _MaterialAppExampleState extends State<MaterialAppExample> {
+  bool isDarkTheme = false;
+  bool customizedThemeAnimation = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      themeMode: isDarkTheme ? ThemeMode.dark : ThemeMode.light,
+      theme: ThemeData(colorScheme: ColorScheme.fromSeed(
+        seedColor: Colors.green,
+      )),
+      darkTheme: ThemeData(colorScheme: ColorScheme.fromSeed(
+        seedColor: Colors.green,
+        brightness: Brightness.dark,
+      )),
+      // When the button with `Customize Theme Animation` text is pressed,
+      // the theme animation will be overridden with the following
+      // [AnimationOverride] instance in the [MaterialApp].
+      themeAnimationOverride: customizedThemeAnimation ? AnimationOverride(
+        curve: Easing.emphasizedDecelerate,
+        duration: const Duration(seconds: 3),
+      ) : null,
```

Comment:
Two trivial suggestions: rename customizedThemeAnimation to isCustomizedThemeAnimation so that it sounds more like a flag than an instance, and put the null `?:` case first to make the expression a tiny bit easier to read.

Same suggestion for the popup menu example.

```suggestion
themeAnimationOverride: !isCustomizedThemeAnimation ? null : AnimationOverride(
  curve: Easing.emphasizedDecelerate,
  duration: const Duration(seconds: 3),
),
```